joomla/joomla-cms
# Updating Composer dependencies

This will update the password_compat library and Symfony YAML Component
### Testing Instructions

The password hashing library is used for our password hashes on PHP 5.3 and 5.4.  So, you'll need to check that logins and password changes still work correctly for these versions of PHP.

The YAML component is supported by `Joomla\Registry\Registry` but outputting objects to YAML format isn't used in core.  If you're really interested in testing this, you could use this test snippet:

``` php
use Joomla\Registry\Registry;

$registry = new Registry;
$registry->set('foo.bar', 'baz');
$registry->toString('YAML');
```
